packages/teleterm/src/ui/services/clusters/clustersService.ts

```diff
@@ -61,24 +61,24 @@ export class ClustersService extends ImmutableStore<ClustersServiceState> {
 
   async syncRootCluster(clusterUri: string) {
     try {
-      await this.syncClusterInfo(clusterUri);
-
-      this.syncLeafClusters(clusterUri);
-      this.syncKubes(clusterUri);
-      this.syncApps(clusterUri);
-      this.syncDbs(clusterUri);
-      this.syncServers(clusterUri);
-      this.syncKubes(clusterUri);
-      this.syncGateways();
+      await Promise.all([
+        this.syncClusterInfo(clusterUri),
+        this.syncLeafClusters(clusterUri),
+      ]);
     } catch (e) {
       this.notificationsService.notifyError({
         title: `Could not synchronize cluster ${
           routing.parseClusterUri(clusterUri).params.rootClusterId
         }`,
         description: e.message,
       });
-      throw e;
     }
+    this.syncKubes(clusterUri);
+    this.syncApps(clusterUri);
+    this.syncDbs(clusterUri);
+    this.syncServers(clusterUri);
+    this.syncKubes(clusterUri);
+    this.syncGateways();
```

Comment:
Are these outside of the `try` block because we're ignoring errors from them anyway?

Reply:
They are outside because each method has its own error handling. Thanks to this we can show a notification from `try catch` and a label above each table.